src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceEducationPreview.vue

```diff
@@ -123,13 +123,7 @@ export default defineComponent({
   methods: {
     formatDate,
     studentFullName(education: Components.Schemas.Transcript) {
-      let fullName = education.studentFirstName;
-      if (education.studentMiddleName) {
-        fullName += ` ${education.studentMiddleName}`;
-      }
-      fullName += ` ${education.studentLastName}`;
-
-      return fullName;
+      return `${education.studentFirstName ?? ""} ${education.studentMiddleName ?? ""} ${education.studentLastName}`;
```

Comment:
hmmm not sure, would this add random spaces to the front? 

ex. 
"  last name"
instead of 
"last name"

might need a trim()?

Reply:
Example
![image](https://github.com/user-attachments/assets/0e058700-9415-4d6e-9c89-4b588c46f10f)

Reply:
Thanks for the review @SoLetsDev . Fixed!